src/Microsoft.AspNet.JsonPatch/Helpers/JsonPropertyMetadata.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.AspNet.JsonPatch.Helpers
+{
+    /// <summary>
+    /// Metadata for JsonProperty.
+    /// </summary>
+    public class JsonPropertyMetadata
```

Comment:
Not sure really what this is accomplishing, name is wierd. I'll come up with a suggestion.